src/Microsoft.Extensions.Http/DependencyInjection/HttpClientFactoryServiceCollectionExtensions.cs

```diff
@@ -15,8 +16,34 @@ public static IServiceCollection AddHttpClient(this IServiceCollection services)
                 throw new ArgumentNullException(nameof(services));
             }
 
+            services.AddOptions();
+
+            services.AddTransient<HttpMessageHandlerBuilder, DefaultHttpMessageHandlerBuilder>();
             services.AddSingleton<HttpClientFactory, DefaultHttpClientFactory>();
             return services;
         }
+
+        public static IServiceCollection AddHttpClient(this IServiceCollection services, string name, Action<HttpClient> configureClient)
```

Comment:
Not exposing an o/l with `Action<HttpClientFactoryOptions>` because the actual options type is pretty gross and not really intended for user code.